Also add $JAVA_HOME/bin to the path if host_os is linux-musl.
--- a/configure.ac
+++ b/configure.ac
@@ -16362,7 +16362,7 @@
 
     case "$host_os" in
 
-    dragonfly*|freebsd*|linux-gnu*|*netbsd*|openbsd*)
+    dragonfly*|freebsd*|linux-gnu*|linux-musl*|*netbsd*|openbsd*)
         if test "$ENABLE_JAVA" != ""; then
             pathmunge "$JAVA_HOME/bin" "after"
         fi
